tests/components/roborock/common.py

```diff
@@ -15,7 +15,7 @@
 from tests.common import MockConfigEntry
 
 
-async def setup_platform(hass: HomeAssistant, platform: str) -> MockConfigEntry:
+async def setup_platforms(hass: HomeAssistant) -> MockConfigEntry:
```

Comment:
Now it only uses a fixture, and it has no parameters anymore; this can be converted into a fixture itself.

Reply:
Changed it into a fixture and into a separate fixture for just creating the config entry when needed